outputs.tf

```diff
@@ -3,10 +3,14 @@
 # ------------------------------------------------------------------------------
 output "arn" {
   description = "The ARN of the certificate that is being validated."
-  value       = "${element(concat(aws_acm_certificate_validation.main.*.certificate_arn, list("")), 0)}"
+  value       = concat(aws_acm_certificate_validation.main.*.certificate_arn, [""])
```

Comment:
`concat(aws_acm_certificate_validation.main.*.certificate_arn, [""])[0]`; you need the `[0]` at the end if not the return type is `list(string)` instead of `string`.